frontend/src/pages/dashboard/organizations/[orgName]/members.tsx

```diff
@@ -28,30 +31,78 @@ const OrganizationMembersPage: NextPage<OrganizationMembersPageProps> = ({
   return (
     <>
       <NextSeo title={`Members - ${organization.title}`} />
-      <Dashboard current="organizations">
-        <div className="px-4 sm:px-6 lg:px-8">
-          <h1 className="text-2xl font-bold tracking-tight sm:text-3xl">
-            Edit organization
-          </h1>
-          <div className="mt-10 max-w-2xl">
-            <OrganizationTabs current="members" />
-            <Button
-              onClick={() => setIsInvitingUser(true)}
-              className="mb-10 ml-auto !inline w-auto"
-              variant="secondary"
-            >
-              Invite member
-            </Button>
+      <Layout>
+<div className="container w-full">
+          <div className="pt-8">
+            <div>
+              <nav aria-label="Back" className="sm:hidden">
+                <button
+                  onClick={() => window.history.back()}
```

Comment:
I think it's better if you use a router push rather than this window.history.back() method because it wouldn't work after a page refresh or if the user came directly to this route.